Documentation/kubernetes-on-baremetal.md

```diff
@@ -0,0 +1,33 @@
+# Kubernetes Installation on Bare Metal &amp; CoreOS
+
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
+After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using he kubectl CLI tool.
+
+## Deployment Prerequisites
+
+### CoreOS Installation
+
+For all nodes running Kubernetes components (master & workers), you must use CoreOS version 773.1.0+ for the kubelet to be present in the image.
+
+Use the official CoreOS Bare Metal guides for installation instructions:
+
+* [Booting with iPXE][coreos-ipxe]
+* [Booting with PXE][coreos-pxe]
+* [Installing to Disk][coreos-ondisk]
+
+[coreos-ipxe]: https://coreos.com/os/docs/latest/booting-with-ipxe.html
+[coreos-pxe]: https://coreos.com/os/docs/latest/booting-with-pxe.html
+[coreos-ondisk]: https://coreos.com/os/docs/latest/installing-to-disk.html
+
+### Kubernetes Pod Network
+
+The following guides assume the use of [flannel][coreos-flannel] as an software-defined overlay network to manage routing of the [pod network][pod-network].
+However, manually managing the Kubernetes network is also an option. See the [Kubernetes networking][kubernetes-networking.md] documentation for more information.
```

Comment:
"Bare Metal is the most common platform where a self-managed network is used, due to the flexibility provided by physical networking gear."